launchable/commands/subset.py

```diff
@@ -247,4 +254,28 @@ def run(self):
 
                 self.print(output)
 
+            build_name, test_session_id = parse_session(session_id)
+            org, workspace = get_org_workspace()
+
+            header = ["", "Candidates",
+                      "Estimated duration (%)", "Estimated duration (min)"]
+            rows = [
+                ["Subset", len(output), summary["subset"]["rate"],
+                 summary["subset"]["duration"]],
+                ["Remainder", len(rests), summary["rest"]
+                 ["rate"], summary["rest"]["duration"]],
+                [],
+                ["Total", len(output) + len(rests), summary["subset"]["rate"] + summary["rest"]
+                 ["rate"], summary["subset"]["duration"] + summary["rest"]["duration"]],
+            ]
+
+            if is_brainless:
+                click.echo(
+                    "INFO: Your model is currently training", err=True)
+
+            click.echo(
+                "Launchable created subset {} for build {} (test session {}) to workspace {}/{}\n".format(subset_id, build_name, test_session_id, org, workspace), err=True)
```

Comment:
Is this something we need to output to stderr? (The one below too)

Reply:
Unfortunately yes... 
Some users save subset list use by pipe `|`. e.g) `launchable subset <ARGS> > subset.txt`
Then, if we print to stdout, these messages will be included subset.txt.

So I printed it to stderr to avoid it.

Reply:
`to workspace` -> `in workspace`

Reply:
fixed https://github.com/launchableinc/cli/commit/ad1edf0b64bf8193224a4f399cd2fd8690d9cdfc

Reply:
Makes sense, thank you!